src/main/java/edu/oakland/production/display02/DisplayCommInterfaceImplementation.java

```diff
@@ -34,6 +37,17 @@ public DisplayCommInterfaceImplementation(DisplayComm2WayInterface displayComm2W
   * @return the TrackData
   */
   public TrackData receiveRfidRequest(int rfidNum) {
-    return comm2Way.passRfidRequest(rfidNum);
+    this.data = comm2Way.passRfidRequest(rfidNum);
```

Comment:
If we remove the above private class attributes, then this gets the type added to it.
```suggestion
    TrackData data = comm2Way.passRfidRequest(rfidNum);
```